tests/base/src/test/java/org/wildfly/security/provider/util/ProviderFactoryTest.java

```diff
@@ -0,0 +1,40 @@
+package org.wildfly.security.provider.util;
```

Comment:
Just minor, this file is missing a copyright header.

Reply:
@fjuma Thank you, fixed.